README.md.erb

```diff
@@ -22,17 +22,16 @@
 ---
 <%- end -%>
 
-Vector is an [open-source][urls.vector_repo] utility for building observability
-pipelines. [Collect][docs.sources], [transform][docs.transforms], and
-[route][docs.sinks] log, metrics and events with one simple tool.
+Vector is a _highly reliable_ observability data router built for demanding
```

Comment:
I'd be careful throwing around "highly reliable" while our version is still sub-1.0 :)